Lesson1/hw1/Main.java

```diff
@@ -0,0 +1,23 @@
+package Lesson1.hw1;
+
+public class Main {
+    public static void main(String[] args) {
+        Object[] array = new Object[3];
+
+        array[0] = new String("Строка");
```

Comment:
I detect that this code is problematic. According to the [Performance (PERFORMANCE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#performance-performance), [Dm: Method invokes inefficient new String(String) constructor (DM_STRING_CTOR)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#dm-method-invokes-inefficient-new-string-string-constructor-dm-string-ctor).
Using the java.lang.String(String) constructor wastes memory because the object so constructed will be functionally indistinguishable from the String passed as a parameter.  Just use the argument String directly.